maxun-core/src/utils/concurrency.ts

```
@@ -3,36 +3,36 @@
-     * Queue of jobs waiting to be completed.
-     */
+  * Queue of jobs waiting to be completed.
+  */
   private jobQueue: Function[] = [];
 
   /**
-     * "Resolve" callbacks of the waitForCompletion() promises.
-     */
+  * "Resolve" callbacks of the waitForCompletion() promises.
+  */
   private waiting: Function[] = [];
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)